src/ImageContainer.js

```diff
@@ -33,18 +47,41 @@ function ImageContainer({ src, title, nextImage, label, bottomText }) {
   }
 
   return (
-    <Flex direction="column" position="relative" minH="100vh" bg="gray.200">
+    <Flex
+      direction="column"
+      alignItems="center"
+      justifyContent="space-evenly"
+      position="relative"
+      minH="100vh"
+      bg="gray.200"
+    >
+      <Heading padding="10px" size="lg">
+        Your score is: {score}
+      </Heading>
       <Flex flex={1} alignItems="center">
-        {!ended && <ImageBlocker {...dimensions} reveal={reveal} />}
+        {!ended && (
+          <ImageBlocker
+            {...dimensions}
+            reveal={reveal}
+            score={score}
+            nextImage={nextImage}
+          />
+        )}
+
         <Image
           src={src}
           alt={title}
           handleDimensionsChange={handleDimensionsChange}
         />
       </Flex>
+      {isCorrect ? <Heading> Boom! You got it!!</Heading> : null}
```

Comment:
maybe something like 
```suggestion
      {isCorrect && <Heading>Correct!</Heading>}
```